### PR TITLE
feat(icons): Add `ethernet-port` and `chevrons-left-right-ellipsis` icons

### LINKED ISSUE #1816: Ethernet or Ethernet port icon
### Icon name

ethernet

### Use cases

I'm working on a network Saas application which offers products related to network switches, ethernet and ethernet ports.

It could also be used by any application that has or manages internet connections, routers, network switches, networking in general etc.

### Design ideas

Screenshot 2024-01-19 at 16 50 23

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

### PR DESCRIPTION
**Pull request by ericfennis**
Closes #1816 

## What is the purpose of this pull request?
- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Icon use case 
To indicate a network connection in a router, network switch, or similar interfaces.
Can also be used as a system icon, next to the existing system icons (wifi, Bluetooth, battery and audio etc)

### Alternative icon designs
See #1816 

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [x] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: 
- [x] I've based them on the following design: 

### Naming 

- x ] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/ethernet-port.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20.2 4H3.8C2.8 4 2 4.7 2 5.6v9.8c0 .9.8 1.6 1.8 1.6H6l3 3h6l3-3h2.2c1 0 1.8-.7 1.8-1.6V5.6c0-.9-.8-1.6-1.8-1.6" />
```
Comment (jguddas) [inline icons/ethernet-port.svg:12]:
```suggestion
  <path d="m15 20 3-3h2a2 2 0 0 0 2-2V6a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2v9a2 2 0 0 0 2 2h2l3 3z" />
```

**Comment by jguddas** [comment]
What about arcifying the eathernet-port icon?

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M10+8v1%22+/%3E%3Cpath+d=%22M14+8v1%22+/%3E%3Cpath+d=%22M14.172+20a2+2+0+0+0+1.414-.586l1.828-1.828A2+2+0+0+1+18.828+17H20a2+2+0+0+0+2-2V6a2+2+0+0+0-2-2H4a2+2+0+0+0-2+2v9a2+2+0+0+0+2+2h1.172a2+2+0+0+1+1.414.586l1.828+1.828A2+2+0+0+0+9.828+20z%22+/%3E%3Cpath+d=%22M18+8v1%22+/%3E%3Cpath+d=%22M6+8v1%22+/%3E">icons<br/>Open lucide studio</a>

**Comment by karsa-mistmere** [comment]
`wired-connection` feels an oddly specific name for an abstract symbol that to my best knowledge is not a standard representation of said concept, so maybe `arrow-double(-headed)-horizontal-dashed` or something similar would be better?

**Comment by ericfennis** [comment]
@karsa-mistmere Kinda true. It's more common than you might think.
image
image

Wiki: https://en.wikipedia.org/wiki/Ethernet
So not sure if we should choose a generic name since it is possibly recognized as a symbol for ethernet. 
But I get you, I still have some doubts about the name. 🤔

**Comment by jguddas** [comment]
> @karsa-mistmere Kinda true. It's more common than you might think. image image
> 
> Wiki: https://en.wikipedia.org/wiki/Ethernet So not sure if we should choose a generic name since it is possibly recognized as a symbol for ethernet. But I get you, I still have some doubts about the name. 🤔

Still feel like it is describing the use case rather than what is actually depicting.

**Comment by ericfennis** [comment]
@jguddas @karsa-mistmere I've renamed it to `chevrons-left-right-ellipsis`

**Pull request merged**